src/errors/index.js

```diff
@@ -0,0 +1,12 @@
+class ValidationError extends Error {
+  constructor(errors) {
+    super()
+    this.name = 'ValidationError'
+    this.type = 'ValidationError'
+    this.errors = errors 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
    this.errors = errors; 
```